### PR TITLE
Added the code to implement doubly linked list using java

### PR DESCRIPTION
**Pull request by cintamg**
Hey, @rudrakshi99 I have written the code.

### CODE REVIEW

**Comment by cintamg** [comment]
Issue #653

**Comment by cintamg** [comment]
@OjusWiZard please review my PR.

File: Code/Java/Doubly_Linked_List.java [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+5.Exit
+Enter your choice
+3
+Enter the position
+2
+Enter the element you want to insert
+88
+
+Output: 23 88 56
+*/
```
Comment (shreyanspoddar) [inline Code/Java/Doubly_Linked_List.java:211]: Add the time and space complexity

File: Code/Java/Doubly_Linked_List.java [hunk 2]
```diff
@@ -0,0 +1,211 @@
+/*
+This is a menu driven (Java)program which has the following choices:
+1. Insert at the beginning.
+2. Insert at the end.
+3. Insert at a specific position.
+4. Delete an element.
+5. Display all the elements in the list.
+*/ 
+
```
Comment (shreyanspoddar) [inline Code/Java/Doubly_Linked_List.java:9]: Elaborate the description

**Review by shreyanspoddar — CHANGES_REQUESTED** [review]
Update the readme

**Comment by cintamg** [comment]
@shreyanspoddar can u reassign this to me.

**Pull request closed without merging**